bcex_core/exchange_interface.py

```diff
@@ -3,8 +3,8 @@
 import math
 import signal
 
-from core.orders import Order, OrderSide, OrderType, TimeInForce
-from core.websocket_client import BcexClient, Book, Channel, Environment
+from bcex_core.orders import Order, OrderSide, OrderType, TimeInForce
```

Comment:
Should we create a folder bcex instead and put the code in it ?

`from bcex.core` seems better than `from bcex_core` 

But agree anyway that it needs to be changed :)

Reply:
ye changed to that